app/services/stories_api/v3/endpoints/story.rb

```diff
@@ -68,7 +68,9 @@ def strip_admin_params
         def story_params
           return params[:story] if params.class == Hash
 
-          params.require(:story).permit(*::StoriesApi::V3::Schemas::Story.rules.keys)
+          # Permitting all params instead of using *::StoriesApi::V3::Schemas::Story.rules.keys
+          # because attributes that are Arrays won't be set correctly (eg. :subjects, instead of subjects: [])
+          params.require(:story).permit!
```

Comment:
Do you know how many keys are there? Can we specify it instead?

Reply:
we do know from the DRY Validation schema, however, I don't want to duplicate them here :(.

There is already a DRY validation in place to validate the attributes further down (https://github.com/DigitalNZ/supplejack_api/blob/master/app/services/stories_api/v3/schemas/story.rb).

Also, when we pass a Hash, we are just sending all the attributes anyway. (https://github.com/DigitalNZ/supplejack_api/pull/194/files#diff-643ae4548cc1072018c34e72ea3e2efeR69)